client/my-sites/plans/current-plan/purchases-listing.jsx

```diff
@@ -124,23 +125,18 @@ class PurchasesListing extends Component {
 	}
 
 	getExpirationInfo( purchase ) {
-		const { translate } = this.props;
-
 		// No expiration date for free plan or partner site.
 		if ( this.isFreePlan( purchase ) || isPartnerPurchase( purchase ) ) {
 			return null;
 		}
 
-		// Show auto-renew information if it is enabled.
-		if ( purchase.autoRenew && purchase.autoRenewDateMoment ) {
-			return translate( 'Set to auto-renew on %s.', {
-				args: invoke( purchase, 'autoRenewDateMoment.format', 'LL' ),
-			} );
```

Comment:
It looks like this logic was removed by mistake?  It seems to have caused the expirational/renewal information on the user's My Plan page to be incorrect...

Tracking interally at pNPgK-4UW-p2

Reply:
Got a pull request up to fix this regression: https://github.com/Automattic/wp-calypso/pull/39951